### ISSUE TITLE
Taxonomies and series

### ISSUE THREAD

**Issue by spanezz** [spanezz/staticsite#21]
(from a conversation with @DonKult)

> > In the example you have a dropdown menu which is nice, but I would like
> > to link to a page showing all entries of the series, which is the page
> > of the associated tag, but reaching that seems hard:
> > url_for(taxonomy('tags').categories[page.meta.series])
> > I was kinda hoping for `url_for(page.meta.series_tag)` especially if
> > – not that I have tried – you have multiple taxonomies with series's…

> If the series is autogenerated from a tag, indeed a way to generate a
> link to the tag page would be enough. If the series is not
> autogenerated from a tag, we currently wouldn't have a recap page for
> the series.
> 
> For the first case, the series feature could easily add a metadata to
> the page pointing to the series tag.
> 
> For the second case, I'm thinking of mandating that series have to be
> tags. One can always create a 'series' taxonomy, if one doesn't want to
> add some series to the normal set of tags. This would also automatically
> provide each series with an RSS feed, and maybe simplify some of the
> series code, by just using tags code to track which pages belong to a
> series.

TL;DR: merge series and taxonomies, mandating the fact that a series is an entry in a taxonomy.

The `series` header can be used to both add a tag and state that it is used as a series.

Document that if one doesn't want to mix series in one's own tag, one can create a new taxonomy for series. That would also give an index pages to all the series in a website.
